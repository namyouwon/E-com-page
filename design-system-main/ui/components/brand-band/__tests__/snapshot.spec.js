/* eslint-env jest */
import React from 'react';
import createHelpers from '../../../../jest.helpers';
import Page from '../';
const { matchesMarkup } = createHelpers(__dirname);

describe('render brand band', () => {
  it('renders small brand band', () => matchesMarkup(<Page size="small" />));
  it('renders medium brand band', () => matchesMarkup(<Page size="medium" />));
  it('renders large brand band', () => matchesMarkup(<Page size="large" />));
  it('renders brand band with no image', () =>
    matchesMarkup(<Page size="none" />));
});
