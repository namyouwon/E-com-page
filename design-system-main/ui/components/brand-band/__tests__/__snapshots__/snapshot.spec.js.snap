// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`render brand band renders brand band with no image 1`] = `
<div
  style={
    Object {
      "backgroundColor": "#eef4ff",
      "minHeight": "20rem",
      "position": "relative",
      "zIndex": "1",
    }
  }
>
  <div
    className="slds-brand-band slds-brand-band_none"
  />
</div>
`;

exports[`render brand band renders large brand band 1`] = `
<div
  style={
    Object {
      "backgroundColor": "#eef4ff",
      "minHeight": "20rem",
      "position": "relative",
      "zIndex": "1",
    }
  }
>
  <div
    className="slds-brand-band slds-brand-band_large"
  />
</div>
`;

exports[`render brand band renders medium brand band 1`] = `
<div
  style={
    Object {
      "backgroundColor": "#eef4ff",
      "minHeight": "20rem",
      "position": "relative",
      "zIndex": "1",
    }
  }
>
  <div
    className="slds-brand-band slds-brand-band_medium"
  />
</div>
`;

exports[`render brand band renders small brand band 1`] = `
<div
  style={
    Object {
      "backgroundColor": "#eef4ff",
      "minHeight": "20rem",
      "position": "relative",
      "zIndex": "1",
    }
  }
>
  <div
    className="slds-brand-band slds-brand-band_small"
  />
</div>
`;
